### ISSUE TITLE
Kaspersky Rescue Disk Updater Error

### ISSUE THREAD

**Issue by tkatsageorgis** [bbalegere/Kaspersky-Rescue-Disk-Updater#5]
Hello there,

after the end of support for Kaspersky Rescue Disk 10 (https://support.kaspersky.com/14540), 
I tested your utility (http://bharat.balegere.com/projects/Kaspersky-Rescue-Disk-Updater/) 
using the Kaspersky Rescue Disk 2018 (https://support.kaspersky.com/viruses/krd2018), 
but unfortunately I got an error message (The Kaspersky Rescue Disk is not a valid Disk. It does not contain the bases folder.)

Can you please help me?

Thanks in advance,
Thanassis

**Comment by Tmp341** [comment]
This repo hasn't been updated for years and Kaspersky created that new Disk version with differences probably.

**Comment by Rosenstein** [comment]
Yes, Kaspersky Rescue Disk 18 is totally different from Kaspersky Rescue Disk 10, and so this repo cannot be used to update virus defs in KRD18.
EDIT:
It seems KRD18 can be updated? without downloading new .iso every time.
http://rescuedisk.kaspersky-labs.com/rescuedisk/updatable/2018/bases/

**Comment by Rosenstein** [comment]
You can now use this repo https://github.com/Rosenstein/Kaspersky-Rescue-Disk18-Updater to download updates for KRD18.

**Comment by tkatsageorgis** [comment]
Thanks a lot for the feedback! I will try it and let you know ASAP!!! ;)
